tests/test_cython/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if python3 -c "import cython" 2>/dev/null; then
```

Comment:
oops, that should be if !python3 -c "import cython"

Reply:
Thanks!